### DISCUSSION TITLE
Tagging help: convert "s01e01" to %disc% %track%

### DISCUSSION THREAD

**Discussion by oshunluvr** [puddletag/puddletag#635] [Q&A]
I record TV shows and the filename structure is: "Showname - s01e01 - EpisodeName", where "s01" means Season 1 and "e01" means Episode 1.

I am trying to tag them using %disc% as the season number and %track% as the episode number but having no luck. I've tried:

%artist% - s$num(%disc%,2)e$num(%track%,2) - %title%

thinking this would skip the spaces, dashes, and the letters "s" and "e" but it doesn't work. I also tried the $validate command but no luck there either. Suggestions?

**Comment by BitterScissors** [comment]
%disc% doesn't work - neither does disk, discnumber, or disknumber.

**Comment by BitterScissors** [comment]
Correction: to get disk number you must use %Disk% (note capitalization).  
I found this under Edit/Extended Tags

**Comment by sandrotosi** [comment]
since these are tv shows, are you sure you wouldnt be better served by series-specific tools, like [tvnamer](https://github.com/dbr/tvnamer) for example?

**Comment by oshunluvr** [comment]
Maybe, thanks for the tip.

**Comment by oshunluvr** [comment]
I'm not using puddle tag to rename files which is what it appears tvnamer
does. I want to create proper tags from existing filenames. I'm not having
trouble naming the files.
